src/cookbooks/dau.md

```diff
@@ -93,3 +94,49 @@ GROUP BY
 ORDER BY
   submission_date_s3
 ```
+
+## Mobile Products
+
+The concept of usage is slightly different for mobile products compared to desktop Firefox.
+A single session of desktop Firefox is likely to span multiple days and we
+rely on a process to send a `main` ping once per day.
+A single session of a mobile application is likely to last only a few minutes and
+we have generally instrumented mobile applications to send a separate ping for
+each user session:
+
+* `core` pings are the canonical measure for usage on legacy mobile products
+* `baseline` pings are the canonical measure for usage on mobile products using the Glean SDK
+
+A given client is considered "active" for a given mobile product on a given day if we receive at
+least one of the above pings. Otherwise, the definitions of DAU and MAU for individual mobile products
+are identical to those used for desktop Firefox.
+
+_Note:_ As of March 2020, Fenix (the new Firefox for Android) is using a modified definition of usage
+which considers a user active for a given day based on any `baseline` _or_ `metrics` ping
+being submitted on the given day. There is an open
+[proposal for Fenix KPI reporting changes](https://docs.google.com/document/d/1Ym4eZyS0WngEP6WdwJjmCoxtoQbJSvORxlQwZpuSV2I/edit?ts=5e6f894f#) to move Fenix reporting to consider only `baseline` pings.
+
+For quick analysis, use the `firefox_nondesktop_exact_mau28_by_dimensions`.
+This table has a `product` dimension used to differentiate different applications
```

Comment:
I have heard many people voice a need for a map between different ways of labeling products.  It gets very confusing across stakeholders.

Reply:
I agree this is a problem. This may be a place where GUD is now the _de facto_ source of truth and we should probably figure out how to make that name translation for broadly available.

Reply:
Recorded in https://github.com/mozilla/bigquery-etl/issues/861